core/src/main/scala/org/http4s/Sigma.scala

```diff
@@ -0,0 +1,15 @@
+package org.http4s
```

Comment:
Maybe into util, the "things that aren't HTTP and that we haven't bothered to make into a separate library" package of shame?

Reply:
Very reasonable.